open-metadata-implementation/integration-services/search-integrator/search-integrator-server/src/main/java/org/odpi/openmetadata/integrationservices/search/contextmanager/SearchIntegratorContextManager.java

```diff
@@ -102,7 +102,14 @@ public void createClients() throws InvalidParameterException {
         try {
             assetCatalogEventClient.registerListener(localServerUserId, eventListener);
         } catch (ConnectionCheckedException | ConnectorCheckedException | PropertyServerException | UserNotAuthorizedException e) {
-            e.printStackTrace();
+            /*
+            The following snippet turns off the Sonar Security Hotspot alert.
+            Sonar considers the printStackTrace as releasing debugging code
+            The Exception copy is needed as a local variable to scope the @SuppressWarnings locally
+             */
+            @SuppressWarnings("java:S4507")
+            Exception error = e;
+            error.printStackTrace();
```

Comment:
As this is not test code I do not think we should be issuing the stack trace - it should go through the usual audit logging of errors.